src/glyph/builder.rs

```diff
@@ -271,6 +282,12 @@ pub struct OutlineBuilder {
     scratch_state: OutlineBuilderState,
 }
 
+#[derive(Debug, Clone, Default, PartialEq)]
+pub struct Outline {
```

Comment:
If this is now purely an internal type, we can remove the `pub`.

Reply:
Hm, then I can't use it in parse.rs :thinking:

Reply:
Removed it from the public API.